src/js/components/award/financialAssistance/CFDAViz.jsx

```diff
@@ -133,6 +142,61 @@ export default class CFDAViz extends React.Component {
         return null;
     }
 
+    chart = () => {
+        const {
+            view,
+            awardTotalObligation,
+            cfda,
+            allCFDAs
+        } = this.props;
+        const awardTotalObligationUnits = calculateUnitForSingleValue(awardTotalObligation, 1);
+        const numeratorTitle = 'Total Funding From This CFDA Program';
+        const denominatorTitle = 'Total Funding From All CFDA Programs';
+        const numerator = {
+            rawValue: cfda._federalActionOblicationAmount,
+            value: cfda.federalActionOblicationAmountShort,
+            text: numeratorTitle
+        };
+        const denominator = {
+            rawValue: awardTotalObligation,
+            value: `${formatMoneyWithPrecision((awardTotalObligation / awardTotalObligationUnits.unit), 1)}${awardTotalObligationUnits.unitLabel}`,
+            text: denominatorTitle
+        };
+        if ((view === 'single' || !view) && (allCFDAs.length > 1)) {
+            if (
```

Comment:
This is a complex conditional. I prefer to store it in a variable like

```
const isConditionMet = (
    condition1.prop2 === someVar ||
    (condition3.prop3 !== someVar && xyz)
)
```

Especially helpful if you can give the var a good name to indicate the condition you're handling.